src/data/queries/submissionStatus.js

```diff
@@ -0,0 +1,66 @@
+import { GraphQLID as ID, GraphQLNonNull as NonNull } from 'graphql';
+import SubmissionStatusType from '../types/SubmissionStatusType';
+import { Vote, Submission, User } from '../models';
+
+export const submissionResult = async submissionId => {
+  const BOARD_MEMBERS = 5;
```

Comment:
I don't think so. Legally the board is 5 members and it feels a little brittle to read from the DB and additionally assert that there are only 5 reviewers.